sphinx/texinputs/Makefile_t

```diff
@@ -7,34 +7,35 @@ ALLPS  = $(addsuffix .ps,$(ALLDOCS))
 
 # Prefix for archive names
 ARCHIVEPRREFIX =
-# Additional LaTeX options
+# Additional LaTeX options.
 LATEXOPTS =
 # format: pdf or dvi
 FMT = pdf
 
-LATEX = latex
-PDFLATEX = {{ latex_engine }}
-MAKEINDEX = makeindex
+LATEX = latexmk -dvi
+PDFLATEXCMD = {{ latex_engine }} --halt-on-error --interaction=nonstopmode
+PDFLATEX = latexmk -pdflatex="$(PDFLATEXCMD) %O %S" -pdf -dvi- -ps-
```

Comment:
unfortunately ``%%O`` and ``%%S`` cause error on mac os x.

I tried various things, based on Latexmk manual, for example using the `-e` option, but I have always have the problem of diverging behaviour of quoting. I can get things working out on mac os x but I think they would not on Windows.

As a result I must find a way to put all into `latexmkrc` file which is handled by Perl.